roles/install-devstack/tasks/main.yml

```diff
@@ -27,6 +27,14 @@
     cmd: |
       set -e
       set -x
+      # As openstack/requirements already upgrade requests to 2.22.0,
+      # but didn't provider a 2.22.0 version for python34 which we used.
+      # Install python3.5
+      add-apt-repository ppa:deadsnakes/ppa -y
+      apt-get update
```

Comment:
do we also need the sudo in here? I didn't see any `become: yes` in here

Reply:
Yeah, if we use sudo here may make this code stronger. But this is a role, the sudo priority should be setted in a job, maybe from a playbook. So here, we need to use become:yes in there.

Reply:
To be honest, use sudo would be good for our code, but see other code in this file, they didn't use sudo, such as iptables need sudo. So that must be another PR to do refact.